lib/gui/os/dropzone/directives/dropzone.js

```diff
@@ -41,7 +41,7 @@ module.exports = ($timeout) => {
       osDropzone: '&'
     },
     link: (scope, element) => {
-      const domElement = element[0];
+      const domElement = _.first(element);
```

Comment:
@jhermsmeier Its a jQuery lite object, in which you can access the native DOM element by accessing its first element (which I agree its weird)